extensions/pipe-commands/src/pipe-to-command.tsx

```diff
@@ -43,7 +43,7 @@ export function PipeCommands(props: { input: PipeInput }): JSX.Element {
       {props.input.type == "file" ? (
         <List.Item
           icon={Icon.Document}
-          title="Open With"
+          title="Open with..."
```

Comment:
I think it's better to name it "Open With…", as this is the way Raycast name this action in the built-in File Search extension.